### DISCUSSION TITLE
2023-09-12

### DISCUSSION THREAD

**Discussion by yeopto** [woowacourse-teams/2023-haru-study#491] [daily_meeting]
### 오늘의 할 일
오전 : 강의
점심 이후 - sse

백엔드 - 인프라개선
프론트 - 에러처리

### 개인 일정
노아 5시 근로회의

### 내일의 할 일
백엔드 - DB 스키마
백엔드 - 인프라 개선하기 과제 업로드
프론트 - 각자 에러 정책 적용

**Comment by aak2075** [comment]
### 체크인

10점 - 해야할 일이 많아 행복하네요

### 체크아웃

10점 - 일들이 마무리가 되어가는 모습이 좋습니다

**Comment by woosung1223** [comment]
### 체크인
7점 - 오늘 할 게 많을 듯 하여 -3점

### 체크아웃
5점 - 정신이 너무 몽롱 하니 집중이 안되네요 ㅠ

**Comment by jaehee329** [comment]
### 체크인
7점 - 화요일인데 체력이 떨어지네요 벌써
### 체크아웃
6점 - 체력 1만큼 감소

**Comment by yeopto** [comment]
### 체크인

7점 - 피곤하네요

### 체크아웃

8점 - 에러 정책이 확정되어서 좋습니다.

**Comment by woo-jk** [comment]
### 체크인
9점 - 나쁘지 않은 하루

### 체크아웃
10점 - 우리의 코드가 점점 완성도 있어지는 모습에 기분이 좋다.

**Comment by nlom0218** [comment]
### 체크인

7점 - 쏘우쏘우

### 체크아웃

5점 - 점점 테코톡이 다가오네요 😭

**Comment by MoonJeWoong** [comment]
### 체크인
6점 - 잠을 잘 못자서 눈이 피로하네요...ㅠㅠ

### 체크아웃
6점 - 할거 너무 많아서 더 덜어질 뻔 했는데 오늘 팀에서 하기로 한 일들은 다 한 것 같아서 유지입니다...
